Develop an SQL schema for SOPs

```diff
@@ -450,39 +685,63 @@ CREATE SEQUENCE public."STUDY_DESIGN_DESCRIPTORS_Design_Descriptors_ID_seq"
     CACHE 1;
 
 
-ALTER TABLE public."STUDY_DESIGN_DESCRIPTORS_Design_Descriptors_ID_seq" OWNER TO postgres;
+ALTER TABLE public."MzTab-M_Small_Molecule_Feature_(SMF)_Section_SMF_ID_seq" OWNER TO postgres;
 
 --
--- TOC entry 2958 (class 0 OID 0)
--- Dependencies: 216
--- Name: STUDY_DESIGN_DESCRIPTORS_Design_Descriptors_ID_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: postgres
+-- TOC entry 3457 (class 0 OID 0)
+-- Dependencies: 239
+-- Name: MzTab-M_Small_Molecule_Feature_(SMF)_Section_SMF_ID_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: postgres
 --
 
-ALTER SEQUENCE public."STUDY_DESIGN_DESCRIPTORS_Design_Descriptors_ID_seq" OWNED BY public."STUDY_DESIGN_DESCRIPTORS"."Design_Descriptors_ID";
+ALTER SEQUENCE public."MzTab-M_Small_Molecule_Feature_(SMF)_Section_SMF_ID_seq" OWNED BY public."MzTab-M_Small_Molecule_Feature_(SMF)_Section"."SMF_ID";
 
 
 --
--- TOC entry 219 (class 1259 OID 17562)
--- Name: STUDY_FACTORS; Type: TABLE; Schema: public; Owner: postgres
+-- TOC entry 237 (class 1259 OID 18396)
+-- Name: MzTab-M_small_molecule_section; Type: TABLE; Schema: public; Owner: postgres
 --
 
-CREATE TABLE public."STUDY_FACTORS" (
-    "Study_Factors_ID" integer NOT NULL,
-    "Factor_Name" text,
-    "Factor_Type" text,
-    "Factor_Type_Term_Accession_Number" text,
-    "Factor_Type_Term_Source_REF" text
+CREATE TABLE public."MzTab-M_small_molecule_section" (
+    "SML_ID" integer NOT NULL,
+    "SMF_ID_REFS_number" integer,
+    "SMF_ID_REFS" integer,
+    database_identifier_number integer,
+    database_identifier text,
+    chemical_formula_number integer,
+    chemical_formula text,
+    smiles_number integer,
+    smiles text,
+    inchi_number integer,
+    inchi text,
+    chemical_name_number integer,
+    chemical_name text,
+    uri_number integer,
+    uri text,
+    theoretical_neutral_mass_number integer,
+    theoretical_neutral_mass double precision,
+    adduct_ions_number integer,
+    adduct_ions text,
+    reliability text,
+    best_id_confidence_measure text,
+    best_id_confidence_value double precision,
+    abundance_assay_number integer,
+    abundance_assay double precision,
+    abundance_study_variable_number integer,
+    abundance_study_variable double precision,
+    abundance_variation_study_variable_number integer,
+    abundance_variation_study_variable double precision,
+    "mzTab-ID_REFS" text
```

Comment:
I had suggest a UML diagram for visualizing the relations between all of these tables previously.  Have you had a chance to look into that?

It would be good to have either a visual summary or some other means to summarize what are the constraints, sequences, unique columns, primary columns, etc. on each table.  Or if you can stick to a standardized naming schema (e.g., REF for foreign keys, and ID for internal or external ids) that would make things much easier to understand.

Reply:
I am looking on it. It looks very cool. Thank you